site/content/getting-started/_index.md

```diff
@@ -14,57 +13,29 @@ The quickstart guide uses all default settings. No additional configuration is r
 
 # What is Contour?  
 
-Contour is an open source Kubernetes ingress controller providing the control plane for the Envoy edge and service proxy.​ Contour supports dynamic configuration updates and multi-team ingress delegation out of the box while maintaining a lightweight profile.  
-
-# Philosophy
-- Follow an opinionated approach which allows us to better serve most users
-- Design Contour to serve both the cluster administrator and the application developer
-- Use our experience with ingress to define reasonable defaults for both cluster administrators and application developers.
-- Meet users where they are by understanding and adapting Contour to their use cases  
-
-See the full [Contour Philosophy][5] page.
+Contour is an Ingress controller for Kubernetes that works by deploying the [Envoy proxy][26] as a reverse proxy and load balancer.
+Contour supports dynamic configuration updates out of the box while maintaining a lightweight profile.
 
-# Why Contour?
-Contour bridges other solution gaps in several ways:
-- Dynamically update the ingress configuration with minimal dropped connections
-- Safely support multiple types of ingress config in multi-team Kubernetes clusters
-- Improve on core ingress configuration methods using our HTTPProxy custom resource
-- Cleanly integrate with the Kubernetes object model  
```

Comment:
we might want to move this content to a different page, will take a look to see if there's another place it makes sense to go..

Reply:
Eh, I think a lot of this is covered in other pages already (e.g. Contour Philosophy which is the 2nd page listed in the docs TOC). Maybe we should add a "Why Contour?" page to docs under the "Introduction" section which could cover some of that, but doesn't need to be tied to this PR.